examples/tweet_generator.py

```diff
@@ -0,0 +1,147 @@
+import re
+import textwrap
+from datetime import datetime
+from PIL import Image, ImageFont, ImageDraw, ImageOps
+from pincer import command, Client, Descripted
+from pincer.objects import Message, Embed, MessageContext
+
+
+class Bot(Client):
+
+    @Client.event
+    async def on_ready(self):
+        print(
+            f"Started client on {self.bot}\n"
+            "Registered commands: " + ", ".join(self.chat_commands)
+        )
+
+    @command(
+        name='twitter',
+        description='to create fake tweets',
+        guild=690604075775164437
+    )
+    async def twitter(self, ctx: MessageContext, content: Descripted[str, '...']):
+        await ctx.interaction.ack()
+
+        message = content
+        for text_match, user_id in re.findall(re.compile(r"(<@!(\d+)>)"), message):
+            message = message.replace(text_match, '@' + str(await self.get_user(user_id)))
+
+        if len(message) > 280:
+            return 'A tweet can be at maximum 280 characters long'
+
+        # wrap the message to be multi-line
+        message = textwrap.wrap(message, 38)
+
+        # download the profile picture and convert it into Image object
+        avatar = await self.get_user(ctx.author.user.id)
+        avatar = await avatar.get_avatar()
+        avatar = avatar.resize((128, 128))
+
+        # modify profile picture to be circular
```

Comment:
maybe extract the full tweet creation to a other method to shorter the command one